lib/cc/engine/analyzable_files.rb

```diff
@@ -19,7 +18,7 @@ def all
       def fetch_files(paths)
         paths.map do |path|
           if path =~ %r{/$}
-            Dir.glob("#{path}/**/*.coffee")
+            Dir.glob("#{path}**/*.coffee")
```

Comment:
I'm not sure. Just looking at this code in isolation it seems pretty broken: this branch is only taken if `path` ends in a slash, so `#{path}/` is clearly wrong. And that sounds exactly like the `.//` bug you describe.